### DISCUSSION TITLE
snyk vs CVE

### DISCUSSION THREAD

**Discussion by alex-at-cascade** [community/community#102660] [Code Security] [answered]
### Select Topic Area

Question

### Body

Will dependabot act on vulnerabilities that don't have a CVE?  E.g. https://security.snyk.io/vuln/SNYK-JS-AXIOS-6144788

**Comment by carogalvin** [comment, marked as answer]
Dependabot will act on any vulnerability that has been submitted and reviewed in the GitHub Advisory Database: https://github.com/advisories

You can learn more about the sources that the GitHub Advisory Database pulls from here: https://docs.github.com/en/code-security/security-advisories/working-with-global-security-advisories-from-the-github-advisory-database/about-the-github-advisory-database

That particular vulnerability you linked is not currently in the Advisory Database.

**Comment by alex-at-cascade** [comment]
That page was helpful, thank you.  As recommended on that page, I opened an issue to include another source: https://github.com/github/advisory-database/issues/3449
